packages/calcite-components/src/components/select/select.e2e.ts

```diff
@@ -22,7 +22,7 @@ describe("calcite-select", () => {
   `;
 
   describe("renders", () => {
-    renders(simpleTestMarkup, { display: "flex" });
+    renders(simpleTestMarkup, { display: "inline" });
```

Comment:
The host display is `flex`, so this test should be failing. Can you double check?